src/discussions/learners/LearnersView.jsx

```diff
@@ -57,10 +57,11 @@ function LearnersView({ intl }) {
   };
 
   return (
-    <div className="d-flex flex-column border-right border-light-300 h-100">
+    <div className="d-flex flex-column border-right border-light-400">
       { !usernameSearch && <LearnerFilterBar /> }
+      <div className="border-bottom border-light-400" />
       { usernameSearch && <SearchInfo text={usernameSearch} count={learners.length} loadingStatus={loadingStatus} onClear={() => dispatch(setUsernameSearch(''))} /> }
-      <div className="list-group list-group-flush learner">
+      <div className="list-group list-group-flush learner" role="list">
```

Comment:
why add the role= list here? is that tested for accessibility ripples?

Reply:
The learner stats cards list is a list. So I added this role for accessibility just as it is used in the post summary card list area.